ParquetClassLibrary/Sandbox/IDs/ChunkType.cs

```diff
@@ -1,8 +1,11 @@
-namespace ParquetClassLibrary.Sandbox
+namespace ParquetClassLibrary.Sandbox.IDs
 {
     /// <summary>
     /// Indicates a the sorts of parquets to generate in a MapChunk.
     /// </summary>
+    // TODO:  This needs revision.  Many more combos of floors and blocks are possible.  (Should we
```

Comment:
How large is a chunk? You could just say, "This chunk is a BeachWater". Then when populating the area, it knows to assess if it's a North, South, etc. facing chunk. Or there could be a ChunkType variable containing Earth and Clouds and River, a facing variable that contains north and south and east, and a cavern-ness variable containing elevation information. Is there a motivation for keeping it all as one enumerator?

Reply:
Oooooooooooo, I really love your second suggestion here!  I will likely do this!

Reply:
Created an issue #58 .